ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/HomeScreenMapContainerFragment.kt

```diff
@@ -128,6 +130,29 @@ class HomeScreenMapContainerFragment : AbstractMapContainerFragment() {
     super.onViewCreated(view, savedInstanceState)
     setupMenuFab()
     setupBottomLoiCards()
+    lifecycleScope.launch {
+      mapContainerViewModel.activeSurvey
+        .combine(mapContainerViewModel.lois) { x, y -> Pair(x, y) }
+        .collect {
+          val survey = it.first
```

Comment:
I thing you can more idiomatically write `.collect{ (survey, lois) -> ..` here

Reply:
yep, good callout